backend/src/org/commcare/suite/model/StackFrameStep.java

```diff
@@ -86,19 +87,17 @@ public boolean getValueIsXPath() {
         return valueIsXpath;
     }
 
-    public void addExtra(String key, String value) {
+    /**
+     * @param value Must extend Externalizable or be a basic data type (String, Vector, etc)
```

Comment:
Yeah, I wanted to take this route, but `<T extends Externalizable>` doesn't work with basic data types